src/legacy/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -59,7 +54,13 @@ export const dateHistogramBucketAgg = new BucketAggType({
     date: true
   },
   makeLabel: function (agg) {
-    const output = this.params.write(agg);
+    let output = '';
+    try {
+      output = this.params.write(agg);
+    } catch (e) {
+      // if interval is invalid, return empty string
```

Comment:
What can cause a write failure? Is it always a good idea to swallow the error? Should we log or do something to indicate a failure here? (I don't know the answers to these questions. I just want to be sure this is what we really want to do. Silent catches are often a source of bugs.)

Reply:
Essentially a write failure is caused by an invalid time interval. But in fact the reason is that this [`makeLabel`](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/series.js#L52) method is invoked every digest cycle. 
On master when a user types an invalid interval, TypeError is printed in console.log. Since currently this error doesn't break UI, and to not break some logic that might be based on such exception, I will leave the code as it is. (I reverted the changes)